library/kong

```diff
@@ -1,13 +1,25 @@
 Maintainers: Kong Docker Maintainers <team-gateway@konghq.com> (@team-gateway-bot)
 GitRepo: https://github.com/Kong/docker-kong.git
 
-Tags: 3.0.0-alpine, 3.0.0, 3.0, alpine, latest
+Tags: 3.0.1-alpine, 3.0.1, 3.0, alpine, latest
+GitCommit: b2057cd3432726c2122873a3740e31a68a03cecd
+GitFetch: refs/tags/3.0.1
+File: Dockerfile.apk
+Architectures: amd64, arm64v8
+
+Tags: 3.0.1-ubuntu, 3.0-ubuntu, ubuntu
+GitCommit: b2057cd3432726c2122873a3740e31a68a03cecd
+GitFetch: refs/tags/3.0.1
+Directory: ubuntu
+Architectures: amd64, arm64v8
+
+Tags: 3.0.0-alpine, 3.0.0
```

Comment:
```suggestion
Tags: 3.0.0-alpine
```
Is this what you were thinking @yosifkit ? I think the reason I left it behind is we always have from the looks of it ...

Reply:
`3.0.0-alpine` and `3.0.0-ubuntu` are the images and tags I was referring to (though I now noticed that there are `2.8.1*` versions to which this comment would also apply). We recommend that the tags kept in the library file are the set of tags that are supported by the upstream (i.e. a new version bump would be released to fix bugs, CVEs, etc). So, when I see a version with a new release that is just patch releases newer (e.g. `3.0.1`), then I usually assume that the older release (`3.0.0`) is no longer supported. From what I've seen, semver based projects don't release sub-patch version bumps and usually focus effort to a limited set of `{major}.{minor}` branches with the latest patch of each `{major}.{minor}` series being the supported version.

Reply:
OK. Reduced down to the most recent patch versions we support so

Instead of 3.0.0, and 3.0.1 just 3.0.1 as well as the 2.8